hat/assets/js/apps/Iaso/domains/forms/components/FormVersionsComponent.js

```diff
@@ -41,6 +41,25 @@ const FormVersionsComponent = ({
             >
                 <FormattedMessage {...MESSAGES.versions} />
             </Typography>
+            <Box
+                mb={2}
+                justifyContent="flex-end"
+                alignItems="center"
+                display="flex"
+            >
+                <FormVersionsDialog
+                    formId={formId}
+                    periodType={periodType}
+                    titleMessage={MESSAGES.createFormVersion}
+                    renderTrigger={({ openDialog }) => (
+                        <AddButtonComponent
+                            onClick={openDialog}
+                            message={MESSAGES.createFormVersion}
+                        />
+                    )}
+                    onConfirmed={() => setForceRefresh(true)}
```

Comment:
Are you sure this is still necessary ? SingleTable is watching `extraProps` normally

Reply:
I don't know it looks like the code of another PR. I just moved the button up honestly